enterprise/server/util/oci/oci.go

```diff
@@ -232,7 +239,52 @@ func NewResolver(env environment.Env) (*Resolver, error) {
 		}
 		allowedPrivateIPNets = append(allowedPrivateIPNets, ipNet)
 	}
-	return &Resolver{env: env, allowedPrivateIPs: allowedPrivateIPNets}, nil
+	imageTagToDigestLRU, err := lru.NewLRU[string](&lru.Config[string]{
```

Comment:
looks like the expiration is not implemented after switching to our `lru` package?

Reply:
thanks for paying attention, i'm cooking it right now, sorry for premature push ;)